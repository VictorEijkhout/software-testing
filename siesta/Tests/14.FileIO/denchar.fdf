%include ../base_wat.fdf
SystemLabel     denchar

Write.Denchar T
DM.UseSaveDM  F
