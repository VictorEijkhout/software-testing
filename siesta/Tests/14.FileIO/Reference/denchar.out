Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/14.FileIO/work/denchar
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:38

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.31881
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel     denchar
Write.Denchar T
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Water molecule
reinit: -----------------------------------------------------------------------
reinit: System Label: denchar
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.560943
                 energy =   -1.731321
                kinetic =    1.587329
    potential(screened) =   -3.318651
       potential(ionic) =  -11.260325

   izeta = 2
                 rmatch =    2.319910
              splitnorm =    0.150000
                 energy =   -1.319217
                kinetic =    2.736746
    potential(screened) =   -4.055963
       potential(ionic) =  -12.745464

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.290924
                 energy =   -0.665565
                kinetic =    4.934810
    potential(screened) =   -5.600376
       potential(ionic) =  -13.346429

   izeta = 2
                 rmatch =    2.782200
              splitnorm =    0.150000
                 energy =   -0.468052
                kinetic =    6.936835
    potential(screened) =   -7.404886
       potential(ionic) =  -15.957492

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.290924
                 energy =    2.236783
                kinetic =    4.278617
    potential(screened) =   -2.041834
       potential(ionic) =   -8.171412
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.290924

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=2  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080

   izeta = 2
                 rmatch =    3.582433
              splitnorm =    0.150000
                 energy =   -0.289774
                kinetic =    1.649095
    potential(screened) =   -1.938868
       potential(ionic) =   -2.516427

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
atom: Total number of Sankey-type orbitals:  5

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   3.561      2.320   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.291      2.782   
   1.000      1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285      3.582   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3

siesta: Automatic unit cell vectors (Ang):
siesta:    7.818555    0.000000    0.000000
siesta:    0.000000    6.218903    0.000000
siesta:    0.000000    0.000000    6.153155

siesta: System type = molecule  

initatomlists: Number of atoms, orbitals, and projectors:      3    23    34

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =    50.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        7.818555    0.000000    0.000000
        0.000000    6.218903    0.000000
        0.000000    0.000000    6.153155

outcell: Cell vector modules (Ang)   :    7.818555    6.218903    6.153155
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    299.1839
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=23 nrows=23 sparsity=1.0000 nnzs=529, refcount: 7>
  <dData1D:(new from dSpData1D) n=529, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=23 nrows=23 sparsity=1.0000 nnzs=529, refcount: 8>
  <dData2D:DM n=529 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      23

InitMesh: MESH = 36 x 30 x 30 = 32400
InitMesh: Mesh cutoff (required, used) =    50.000    58.594 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -80.204204
siesta: Eions   =       810.513381
siesta: Ena     =       174.158273
siesta: Ekin    =       378.896231
siesta: Enl     =       -70.921080
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -21.910068
siesta: DUscf   =         2.666745
siesta: DUext   =         0.000000
siesta: Ex      =      -104.754824
siesta: Ec      =       -12.801811
siesta: Exc     =      -117.556635
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -465.952445
siesta: Etot    =      -465.179916
siesta: FreeEng =      -465.179916

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -465.952445     -465.179916     -465.179916  1.176914 -0.767639  8.301973
timer: Routine,Calls,Time,% = IterSCF        1       0.004   0.37
   scf:    2     -466.839609     -466.161373     -466.161373  0.110847 -6.084304  3.251141
   scf:    3     -466.313402     -466.292858     -466.292858  0.048279 -5.120341  0.338266
   scf:    4     -466.299511     -466.296478     -466.296478  0.010458 -5.049304  0.275184
   scf:    5     -466.299027     -466.298128     -466.298128  0.015839 -4.886674  0.118636
   scf:    6     -466.298135     -466.298135     -466.298135  0.005438 -4.784418  0.029244
   scf:    7     -466.298268     -466.298211     -466.298211  0.001732 -4.795814  0.018244
   scf:    8     -466.298234     -466.298225     -466.298225  0.001073 -4.806284  0.007571
   scf:    9     -466.298226     -466.298225     -466.298225  0.000257 -4.812217  0.001578
   scf:   10     -466.298226     -466.298226     -466.298226  0.000125 -4.811853  0.001378
   scf:   11     -466.298226     -466.298226     -466.298226  0.000054 -4.811701  0.001231
   scf:   12     -466.298226     -466.298226     -466.298226  0.000036 -4.811506  0.000974

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000364764
max |H_out - H_in|      (eV) :     0.0009736142
SCF cycle converged after 12 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      23

siesta: E_KS(eV) =             -466.2982

siesta: E_KS - E_eggbox =      -466.2982

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.040806    0.000000
----------------------------------------
   Max    1.094689
   Res    0.618991    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    1.094689    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -13.48       -3.84        2.37        0.00        0.00        0.00
(Free)E + p*V (eV/cell)     -465.3680
Target enthalpy (eV/cell)     -466.2982

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -106.498977
siesta: Eions   =       810.513381
siesta: Ena     =       174.158273
siesta: Ekin    =       350.848000
siesta: Enl     =       -63.700877
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -5.082794
siesta: DUscf   =         0.849997
siesta: DUext   =         0.000000
siesta: Ex      =      -100.269421
siesta: Ec      =       -12.588022
siesta: Exc     =      -112.857443
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -466.298226
siesta: Etot    =      -466.298226
siesta: FreeEng =      -466.298226

siesta: Final energy (eV):
siesta:  Band Struct. =    -106.498977
siesta:       Kinetic =     350.848000
siesta:       Hartree =     397.001958
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -100.269421
siesta:         Corr. =     -12.588022
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -112.857443
siesta:  Ion-electron =   -1103.481211
siesta:       Ion-ion =       2.190471
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -466.298226
siesta:         Fermi =      -4.811506

siesta: Atomic forces (eV/Ang):
siesta:      1    0.000000   -1.094689    0.000000
siesta:      2    0.895916    0.567748   -0.000000
siesta:      3   -0.895916    0.567748   -0.000000
siesta: ----------------------------------------
siesta:    Tot    0.000000    0.040806    0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.008412    0.000000    0.000000
siesta:     0.000000   -0.002396   -0.000000
siesta:     0.000000    0.000000    0.001480

siesta: Cell volume =        299.183863 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00003387          0.00000952  Ry/Bohr**3
siesta:           0.00310936          0.00087453  eV/Ang**3
siesta:           4.98174341          1.40114954  kBar
(Free)E+ p_basis*V_orbitals  =        -465.585589
(Free)Eharris+ p_basis*V_orbitals  =        -465.585589

siesta: Electric dipole (a.u.)  =    0.000000    0.582579   -0.000000
siesta: Electric dipole (Debye) =    0.000000    1.480768   -0.000000

dhscf: Vacuum level (max, mean) =    0.522173   -0.066639 eV

cite: Please see "denchar.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.079
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.078       1.078   100.00
Setup                  1       0.069       0.069     6.39
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.005       1.005    93.20
geom_init              1       0.956       0.956    88.61
state_init             1       0.108       0.108    10.05
hsparse                1       0.000       0.000     0.01
overlap                1       0.108       0.108    10.03
Setup_H0               1       0.844       0.844    78.26
naefs                  2       0.000       0.000     0.03
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.053       0.106     9.83
nlefsm                 2       0.365       0.731    67.76
DHSCF_Init             1       0.008       0.008     0.70
DHSCF1                 1       0.001       0.001     0.10
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.006       0.006     0.59
REMESH                 1       0.000       0.000     0.05
REORD                 37       0.000       0.001     0.07
PHION                  1       0.004       0.004     0.40
COMM_BSC              62       0.000       0.001     0.07
POISON                16       0.000       0.008     0.74
fft                   32       0.000       0.006     0.52
setup_H               13       0.003       0.040     3.69
DHSCF                 15       0.004       0.054     5.03
DHSCF3                15       0.003       0.047     4.37
rhoofd                15       0.001       0.010     0.96
XC                    15       0.001       0.016     1.53
GXC-CellXC            15       0.001       0.016     1.48
gridxc@cellXC         15       0.001       0.016     1.48
vmat                  14       0.001       0.007     0.69
IterSCF               12       0.003       0.038     3.56
compute_dm            12       0.000       0.001     0.12
diagon                12       0.000       0.001     0.11
r-eigvec              12       0.000       0.001     0.10
r-buildHS             12       0.000       0.000     0.00
rdiag                 12       0.000       0.001     0.10
rdiag1                12       0.000       0.000     0.00
rdiag2                12       0.000       0.000     0.01
rdiag3                12       0.000       0.000     0.04
rdiag4                12       0.000       0.000     0.04
r-buildD              12       0.000       0.000     0.01
MIXER                 11       0.000       0.000     0.01
PostSCF                1       0.011       0.011     1.02
DHSCF4                 1       0.007       0.007     0.66
dfscf                  1       0.006       0.006     0.57
overfsm                1       0.000       0.000     0.01
writeHSX               1       0.000       0.000     0.00
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.004       0.004     0.40
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:39
Job completed
